packages/common/src/lib/address.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Defines the shape of an address object
+ * @class
+ */
+class Address {
+	/**
+	 * @type {string} - the first line of the address
+	 */
+	addressLine1;
+
+	/**
+	 * @type {string} - the second line of the address
+	 */
+	addressLine2;
+
+	/**
+	 * @type {string} - the name of the town, city or other settlement
+	 */
+	town;
+
+	/**
+	 * @type {string} - the name of the county
+	 */
+	county;
```

Comment:
Did we have a separate county field? I thought we just had 4 fields. If so then all good.

Reply:
oops, I probably got this from looking at addresses on the appellant journey - if we don't need county, I will eliminate it